src/views/blocks/ImageTextBlock.php

```diff
@@ -26,5 +29,4 @@
             <?php endif; ?>
         </div>
     </div>
-    <div style="clear:both"></div>
```

Comment:
In general, I think this commit is an improvement from front-end side regarding <div> markup and class names.

some thoughts:

- but there are still  a lot of hardcodes values, can the be removed?
- is the removele of clear:both not breaking the usage in combination with other bs field?

Reply:
- Which hardcode values do you mean? `media-*` classes?
- I think because of the `media` class the `clear:both` is obsolete. Maybe with `clear:both` it  have a better browser support, but I would prefer the`clearfix` class.

Reply:
- the numbers like `20px`
- yes, a clearfix class would be better